lib/dataset.js

```diff
@@ -86,7 +86,7 @@ class Dataset {
             dct:title ?title .
         }
       } WHERE {
-        GRAPH <${this.graph}> {
+        GRAPH <${PUBLIC_GRAPH}> {
           ?meeting a <${VERGADERACTIVITEIT_TYPE}> ;
             besluit:geplandeStart ?start .
           BIND (CONCAT("Vergaderactiviteit in kort bestek ", ?start) AS ?title)
```

Comment:
I think this used to work because the temp graph only contains a single meeting. Because now we do `moveGraph` before, the temp graph is (should be?) empty and we can't rely on that. We should instead get the meeting URI + geplande start before hand and pass it to this function. Maybe it can be a member variable of the dataset class and be set on a per-instance basis.